solidity/contracts/CronosGravity.sol

```diff
@@ -70,17 +80,56 @@ contract CronosGravity is Gravity, AccessControl, Pausable, Ownable {
         _unpause();
     }
 
+    /**
+    * Only owner
+    * Notify the start of the migration
+    */
+    function startMigration() public onlyOwner {
+        migrationHeight = block.number + migrationPeriod;
+        migration = true;
+        emit Migration(true);
+    }
+
+    /**
+    * Only owner
+    * Stop the migration
+    */
+    function stopMigration() public onlyOwner {
+        // add approximately one day to the migration start height
+        migrationHeight = 0;
+        migration = false;
+        emit Migration(false);
+    }
+
+    /**
+    * Only owner
+    * Set a new migration period
+    */
+    function setMigrationPeriod(uint256 _migrationPeriod) public onlyOwner {
```

Comment:
May not be needed if we think that ethereum 2.0 wont be significantly faster than current network.

@FinnZhangCrypto?

Reply:
Ethereum after The Merge will be more consistent at 12 seconds. In short the only different is it will be more predictable height to time.

Reply:
Okay, in that case this method is not necessary